core-bundle/contao/dca/tl_content.php

```diff
@@ -612,14 +612,14 @@
 		'playerStart' => array
 		(
 			'inputType'               => 'text',
-			'eval'                    => array('rgxp'=>'natural', 'nospace'=>true, 'tl_class'=>'w25'),
-			'sql'                     => "int(10) unsigned NOT NULL default 0"
+			'eval'                    => array('maxlength'=>8, 'nospace'=>true, 'tl_class'=>'w25'),
+			'sql'                     => "varchar(8) COLLATE ascii_bin NOT NULL default ''"
```

Comment:
This will lead to an error, if someone has something like 100000000 defined already (however improbable that might be). In addition, 8 characters is not enough for some of the examples from https://www.w3.org/TR/media-frags/#naming-time
Let's increase it to 16 characters may be?

```suggestion
			'eval'                    => array('maxlength'=>16, 'nospace'=>true, 'tl_class'=>'w25'),
			'sql'                     => "varchar(16) COLLATE ascii_bin NOT NULL default ''"
```